pkg/acsengine/defaults_test.go

```diff
@@ -503,6 +503,71 @@ func TestAgentPoolProfile(t *testing.T) {
 	}
 }
 
+// TestSetComponentsNetworkDefaults covers tests for setMasterNetworkDefaults and setAgentNetworkDefaults
+// TODO: Currently this test covers only api.Distro setting. Extend test cases to cover network configuration too.
+func TestSetComponentsNetworkDefaults(t *testing.T) {
+
+	var tests = []struct {
+		name string
+		//orchestrator to be tested
+		api.OrchestratorProfile
+		//expected result default disto
+		api.Distro
+		//test result expectation for false positives (false - fail, true - succ)
+		bool
```

Comment:
give the field a name please

Reply:
call api.Distro `expectedDistro` then test against it and you shouldn't need the bool member at all?

Reply:
bool member is to detect false/positive. If code changes to the point where it gives results where tests should fail, we dont test failures. This allows to test failed code usecases